component/background/StorageService.tsx

```diff
@@ -0,0 +1,92 @@
+import {STORAGE_KEY_ID} from "../client/Constants";
+
+class StorageService {
+  private storage: chrome.storage.StorageArea;
+
+  constructor() {
+    this.storage = chrome.storage.sync;
+    this.storage.get(STORAGE_KEY_ID, (id) => {
```

Comment:
What is STORAGE_KEY_ID for?
Is the store namespace shared among different extensions?